Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Recurring_rare_domain_access_from_an_unsigned_process.yml

```diff
@@ -0,0 +1,933 @@
+      scriptName: SearchIncidentsV2
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      fromdate:
+        simple: 5 hours ago
+      includeinformational:
+        simple: "true"
+      query:
+        simple: cid:${alert.cid.[0]} AND (name:"Unsigned process makes connections over DNS ports" or name:"Scripting engine injects into another process" or name:"Uncommon WPAD quires to a rare domain" or name:"Wildfire Malware" or name:"Impair Defenses - 1497766117")
+      todate:
+        simple: now
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 59.5,
+          "y": 654
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "29":
+    id: "29"
+    taskid: 47c1f3a7-df7d-43f9-9b8c-2ad2b2db84c1
+    type: regular
+    task:
+      id: 47c1f3a7-df7d-43f9-9b8c-2ad2b2db84c1
+      version: -1
+      name: Add Suspicious Domain Monitoring
+      description: send HTTP POST request.
+      script: '|||core-api-post'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "17"
+    scriptarguments:
+      body:
+        simple: |-
+          {
+            "request_data": "indicator,type,severity,expiration_date\n${Core.OriginalAlert.raw_abioc.event.domain},DOMAIN_NAME,MEDIUM,${iocExpiration}","validate":"true"
+          }
+      uri:
+        simple: /public_api/v1/indicators/insert_csv
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 60.5,
+          "y": 1697
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "32":
+    id: "32"
+    taskid: a23c840c-5445-48c8-b061-80eafd2ee18f
+    type: regular
+    task:
+      id: a23c840c-5445-48c8-b061-80eafd2ee18f
+      version: -1
+      name: Calculate Now + 2 weeks in epoch
+      description: "Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.\n\nThis automation runs using the default Limited User role, unless you explicitly change the permissions.\nFor more information, see the section about permissions here:\n- For Cortex XSOAR 6 see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.x/Cortex-XSOAR-Playbook-Design-Guide/Automations \n- For Cortex XSOAR 8 Cloud see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Create-a-script\n- For Cortex XSOAR 8.7 On-prem see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/Create-a-script"
```

Comment:
What about XSIAM?

 For Cortex XSIAM, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Documentation/Create-a-script